data/reusables/dependabot/supported-package-managers.md

```diff
@@ -102,6 +102,7 @@ For more information about using {% data variables.product.prodname_dependabot_v
 
 {% data variables.product.prodname_dependabot %} doesn't run Gradle but supports updates to the following files:
 * `build.gradle`, `build.gradle.kts` (for Kotlin projects)
+* `gradle/wrapper/gradle-wrapper.properties` (for Gradle wrapper)
```

Comment:
@gmazzo Something we need to document as well is that the gradle wrapper dependency is called `[gradle-wrapper ](https://github.com/dependabot/dependabot-core/blob/c938bbf7cb4da88053d4379dcab297a3eaa8c0a7/gradle/lib/dependabot/gradle/file_parser/distributions_finder.rb#L55-L61)` within Dependabot

This is needed to configure the various dependency rules for dependabot

Reply:
Can you make a suggestion on this?